spec/models/user_spec.rb

```diff
@@ -51,6 +51,11 @@
       expect(build(:partner, email: "@boooooooooo")).not_to be_valid
       expect(build(:partner, email: "boooooooooo@")).not_to be_valid
     end
+    it "requires a password with a special character and number" do
+      expect(build(:user, password: "password", password_confirmation: "password")).not_to be_valid
+      expect(build(:user, password: "a;dsfj!55", password_confirmation: "a;dsfj!55" )).to be_valid
+      expect(build(:user, password: "password5", password_confirmation: "password5")).to be_valid
```

Comment:
This only has a number, not a special character.

```suggestion
      expect(build(:user, password: "password5", password_confirmation: "password5")).not_to be_valid
```

Reply:
Oh, thank you. I misread somewhere and thought that it required a number or special character